executor/aggregate_test.go

```diff
@@ -725,3 +725,17 @@ func (s *testSuite) TestIssue10608(c *C) {
 	tk.MustExec("INSERT INTO s VALUES (100292, 508931),(120002, 508932);")
 	tk.MustQuery("select (select group_concat(concat(123,'-')) from t where t.id = b group by t.id)  from s;").Check(testkit.Rows("123-", "123-"))
 }
+
+func (s *testSuiteAgg) TestPR15242ShallowCopy(c *C) {
```

Comment:
Please fix this issue: "executor/aggregate_test.go:729:10: undefined: testSuiteAgg"

Reply:
ok, tks.